salt/files/cloud.providers.d/ec2.conf

```diff
@@ -2,13 +2,13 @@
 {% set cloud = salt['pillar.get']('salt:cloud', {}) -%}
 ec2_ubuntu_public:
   minion:
-    master: {{ cloud['master'] }}
+    master: {{ cloud.get('master', 'salt') }}
```

Comment:
It doesn't use defaults.get, it uses merging in the map.jinja (although that explains where we pulled that default.{yaml,json} stuff from).